Source/SwiftLintFramework/Rules/ForceUnwrappingRule.swift

```diff
@@ -32,7 +33,7 @@ public struct ForceUnwrappingRule: OptInRule, ConfigurationProviderRule {
     )
 
     public func validateFile(file: File) -> [StyleViolation] {
-        return file.matchPattern("[\\w)]+!", withSyntaxKinds: [.Identifier]).map {
+        return file.matchPattern("([\\w)]+!)|(\"\\)!)", withSyntaxKinds: [.Identifier, .String]).map {
```

Comment:
Good point.

Maybe I'll wrangle with the regex until it catches that too. Or maybe it should only be used in conjunction with whitespace rules. :)